tool/teleport/common/teleport.go

```diff
@@ -71,6 +71,8 @@ func Run(cmdlineArgs []string, testRun bool) (executedCommand string, conf *serv
 	start.Flag("debug", "Enable verbose logging to stderr").
 		Short('d').
 		BoolVar(&ccf.Debug)
+	start.Flag("disable-tls", "Disable TLS use").
```

Comment:
Lets do this:

```go
start.Flag("insecure-no-tls", "Disable TLS for the web socket"
```

We always insert "insecure" word into CLI flags we do not recommend for production use.